package.json

```diff
@@ -62,6 +63,14 @@
       "env": {
         "AMO_CDN": "https://addons-dev-cdn.allizom.org",
         "PROXY_API_HOST": "https://addons-dev.allizom.org",
+        "FXA_CONFIG": "default",
```

Comment:
I think you just changed the FXA config of our local setup, which is not nice 😛 

It should be `local` here, not `default`.